test-lib.sh

```diff
@@ -544,4 +545,18 @@ EOF
     eval "$oldstate"
 }
 
+# ct_show_resources
+# ----------------
+# Prints the available resources
+ct_show_resources()
+{
+  echo "Resources info:"
+	echo "Memory:"
```

Comment:
Just a nitpick, use spaces instead of tabs here

Reply:
Fixed in the rebased PR.